src/utils/env.ts

```diff
@@ -23,6 +23,25 @@ export function isAppUniswapStagingOrg({ hostname }: { hostname: string }): bool
   return hostname === 'app.uniswap-staging.org'
 }
 
+export function isBrowserRouterEnabled(): boolean {
+  if (isProductionEnv()) {
+    if (
+      isAppUniswapOrg(window.location) ||
+      isAppUniswapStagingOrg(window.location) ||
+      // Cypress tests
+      isLocalhost(window.location)
+    ) {
+      return true
+    }
+    return false
```

Comment:
can you add a comment explaining when `false` is expected? it's just IPFS right?